### PR TITLE
Off-Design documention

### PR DESCRIPTION
**Pull request by kaushikponnapalli**
### Summary

Added documentation for off design missions.

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/docs/user_guide/off_design_missions.ipynb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Off-Design Missions\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "Reserve missions are enabled for the following mission types:\n",
```
Comment (jkirk5) [inline aviary/docs/user_guide/off_design_missions.ipynb:11]:
```suggestion
    "Off-design missions are enabled for the following mission types:\n",
```

**Review by jkirk5 — APPROVED** [review]

File: aviary/docs/examples/off_design_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    "An overview of off-design missions is presented in [Off-Design Mission User Guide](../user_guide/off_design_missions).\n",
+    "\n",
+    "The example is available in `aviary/examples/off_design_example.py`\n",
+    "The examples start with `run_aviary_example.py` and add off-design missions.\n",
+    "\n",
+    "The results of the sizing problem are saved using `prob.save_sizing_to_json()`.\n",
+    "By default, this method saves the results to `sizing_problem.json` but the name of the file may be changed using the `json_filename` argument.\n",
+    "The saved problem may then be used to run off-design missions using `prob.fallout_mission()` and `prob.alternate_mission()`.\n",
+    "\n",
+    "This example can be modified to use the 2DOF formulation by using `aircraft_for_bench_GwGm` instead of `aircraft_for_bench_FwFm` and import the default two_dof phase_info instead of the `phase_info` declared in the problem.\n"
```
[the remark this answers is not in the document]
Comment (jkirk5) [reply aviary/docs/examples/off_design_example.ipynb:21]: Is the phase_info declared in the file meaningfully different from the default? If not, just import it too. If it is, we should have an example of a unique 2DOF phase_info too, and the user can flip which one is loaded at the function call

File: aviary/docs/user_guide/off_design_missions.ipynb [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    "\n",
+    "* height_energy    (completed)\n",
+    "* 2DOF (collocation) (complete)\n",
+    "* 2DOF (shooting)    (future work)\n",
+    "* height_energy (shooting)    (future work)\n",
```
Comment (crecine) [inline aviary/docs/user_guide/off_design_missions.ipynb:16]: is this accurate?

File: aviary/docs/user_guide/off_design_missions.ipynb [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Off-Design Missions\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "Reserve missions are enabled for the following mission types:\n",
+    "\n",
+    "* height_energy    (completed)\n",
```
Comment (crecine) [inline aviary/docs/user_guide/off_design_missions.ipynb:13]:
```suggestion
    "* height_energy (collocation)    (completed)\n",
```

**Review by crecine — APPROVED** [review]

File: aviary/docs/user_guide/off_design_missions.ipynb (hunk 3 above)
Comment (kaushikponnapalli) [reply aviary/docs/user_guide/off_design_missions.ipynb:16]: Yeah off-design works for height energy and 2DOF collocation. I assumed we will at some point add them into shooting as well but if that's not the case, I'll change it

Comment (crecine) [reply aviary/docs/user_guide/off_design_missions.ipynb]: Sorry, I should have clarified. I think the 2dof shooting also works, and I'm not sure if the height energy shooting works. We should check that.

also, I just noticed that height energy says "completed" and 2dof says "complete", we should make those the same.

**Pull request merged**